doc/doxygen/Makefile

```diff
@@ -12,9 +12,25 @@ ifeq (,$(LESSC))
     LESSC=lessc
   else ifneq (,$(shell command -v lesscpy 2>/dev/null))
     LESSC=lesscpy
+  else
+    $(warning Did not find a LESS compiler, so I cannot build the CSS file in case you modified \
+              the LESS file.)
   endif
 endif
 
+SED=sed
+UNAME := $(shell uname -s)
+ifneq (,$(LESSC))
+    ifeq ($(UNAME),Darwin)
+        ifneq (,$(shell command -v gsed 2>/dev/null))
+            SED=gsed
+        else
+            $(error The version of sed that ships with macOS is very old, so you need gnu-sed. \
```

Comment:
maybe make this a warning and `unset LESSC` to avoid having this fail here, bc it does work for me w/o having `lessc` and `gnu-sed`.

Reply:
`unset LESSC` was not included in the latest fix.